src/routes/accounts/signup.ts

```diff
@@ -18,7 +18,7 @@ router.post(
     // Check if user exists in db.
     const existingUser = await User.findOne({ email });
     if (existingUser) {
-      res.status(400).json({ error: 'user already exists' });
+      res.status(409).json({ error: 'user already exists' });
```

Comment:
Didn't know that 409 is the correct message error for already saved records.

Thanks for pointing this out